xunit-autolabeler-v2/ast_parser/lib/file_utils.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from os import path
+import subprocess
+
+from .constants import REGION_TAG_GREP_ARGS
+
+
+def __getFiles(root_dir, predicate):
```

Comment:
Single underscores instead of dunder for private methods?

Reply:
Done.